src/borg/archiver/compact_cmd.py

```diff
@@ -176,32 +176,33 @@ def build_parser_compact(self, subparsers, common_parser, mid_common_parser):
             Free repository space by deleting unused chunks.
 
             borg compact analyzes all existing archives to find out which repository
-            objects are actually used (referenced). It then removes all unused objects
-            to free repository space.
+            objects are actually used (referenced). It then deletes all unused objects
+            from the filesystem to free space.
 
             Unused objects may result from:
 
             - borg delete or prune usage
-            - interrupted backups (maybe retry the backup first before running compact!)
+            - interrupted backups (maybe retry the backup first before running compact)
             - backup of source files that had an I/O error in the middle of their contents
-              and that were skipped due to this.
-            - corruption of the repository (e.g. the archives directory having lost entries)
+              and that were skipped due to this
+            - corruption of the repository (e.g. the archives directory having lost
+              entries, see notes below)
 
             You usually don't want to run ``borg compact`` after every write operation, but
             either regularly (e.g. once a month, possibly together with ``borg check``) or
             when disk space needs to be freed.
 
             **Important:**
 
-            After compacting it is not possible anymore to use ``borg undelete`` to recover
-            previously deleted archives.
+            After compacting it is no longer possible to use ``borg undelete`` to recover
+            soft-deleted archives.
```

Comment:
except the soft-deleted archives you deleted after that compact. :-) that's why I added "previously".

Reply:
Hmmm... :thinking: Yeah, that's right. The reason I wanna change that is because it kinda implies that it's somehow possible to recover archives that were "truly deleted" before. Let's just leave that "deleted" part out completely, because after `compact` it can't recover anything, so I just changed it to "to recover archives".

Reply:
What I mean is, consider this sequence:
```
D D D C D D ...
```
D is a soft-deletion of an archive, C is compact.

You are saying the after C (to the right of C) it is not possible to recover soft-deleted archives, but it is, but only the one to the right of C. That's why I wanted that "previously".

Reply:
Ah! Sorry, I didn't understand what you meant. That's indeed a valid point, I didn't even think about that users could assume that compaction has an ongoing effect, but nothing really contradicts that. So, "recover previously soft-deleted archives" it is I guess? :smiley:

Reply:
You wouldn't believe what users can "read between the lines". :-)